datasource/http/data_acc_test.go

```diff
@@ -49,6 +52,14 @@ func TestHttpDataSource(t *testing.T) {
 				"error": "the `url` must be specified",
 			},
 		},
+		{
+			Name:  "method_is_invalid",
+			Path:  testDatasourceInvalidMethod,
+			Error: true,
+			Outputs: map[string]string{
+				"error": "the `method` must be one of [HEAD GET POST PUT]",
```

Comment:
With the augmented list of methods, won't this fail? Or if it doesn't maybe there's something wrong with the test.

Reply:
It does a regex match with the actual and expected output via `regexp.MatchString(val+".*", logsString)`. Due to the wildcard end in the regex the test did not fail. I fixed it anyway with the expected values.  
Thanks for pointing out the missed list of supported methods